templates/books-show.html

```diff
@@ -0,0 +1,59 @@
+<!-- header -->
+<div class="jumbotron">
+  <div class="container">
+    <div class="row yay">
+      <div class="col-md-2">
+        <a ng-href="/">
+          <h1><span class="glyphicon glyphicon-arrow-left back-arrow" aria-hidden="true"></span></h1>
+        </a>
+      </div>
+      <div class="col-md-10">
+        <h1>{{booksShowCtrl.book.title}}</h1>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- book/edit form -->
+<div class="container">
+  <div class="form-inline">
+    <div class="row">
+      <!-- image -->
+      <div class="col-md-3 col-md-offset-3">
+        <img class="showpage-image" ng-src="{{booksShowCtrl.book.image}}"></img>
+      </div>
+      <!-- book info/edit form -->
+      <form>
+        <div class="col-md-3 book-page-text">
+          <div class="book-info" ng-hide="editing">{{booksShowCtrl.book.title}}</div>
+          <input ng-show="editing" ng-model="booksShowCtrl.book.title" type="text" class="form-control input-md" placeholder="title">
+
+          <div class="book-info" ng-hide="editing">{{booksShowCtrl.book.author}}</div>
+          <input ng-show="editing" ng-model="booksShowCtrl.book.author" type="text" class="form-control input-md" placeholder="author">
+
+          <div class="book-info" ng-hide="editing">{{booksShowCtrl.book.releaseDate}}</div>
+          <input ng-show="editing" ng-model="booksShowCtrl.book.releaseDate" type="text" class="form-control input-md" placeholder="release date">
+
+          <input ng-show="editing" ng-model="booksShowCtrl.book.image" type="text" class="form-control input-md" placeholder="image">
+          <label ng-show="editing">image url</label>
+
+          <!-- buttons -->
+          <div>
+            <button class='btn btn-default book-info' ng-click="booksShowCtrl.deleteBook(booksShowCtrl.book)" aria-label="delete">
+              <span class="glyphicon glyphicon-trash" aria-hidden="true"></span>
+            </button>
+            <button class='btn btn-default book-info' ng-hide="editing" ng-click="editing = true" aria-label="edit">
```

Comment:
Everywhere that you're setting editing using 2 lines of JS in your ng-click, or like here, it would be better to put that in a variable inside of booksShowCtrl and include this.editing=true in the other functions you call as click callbacks.